src/app/base/components/AppSideNavigation/_index.scss

```diff
@@ -74,6 +74,7 @@
             display: inline-block;
             width: 100%;
             height: 100%;
+            text-align: left;
```

Comment:
What made this necessary?

Reply:
By default, the logout button aligns text to the center, so I had to add this to ensure it aligns to the left